### PR TITLE
Currency: Added ubi:9.3 build_script,Dockerfile and updated build_info.json for glassfish

### PR DESCRIPTION
**Pull request by ramnathnayak-ibm**
The job on travis is exceeding the maximum log length, and hence is getting terminated. Attaching the log file where build and test is successfull.
[glassfish_7.0.15_logs.txt](https://github.com/user-attachments/files/16427827/glassfish_7.0.15_logs.txt)

## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: g/glassfish/glassfish_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# Set MAVEN_OPTS environment variable
+export MAVEN_OPTS="-Xmx2500m -Xss768k -XX:+UseG1GC -XX:+UseStringDeduplication"
+
+# Clone package repository
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+# Build, clean and test
+if ! mvn -B -e clean install -Pfastest,staging -T4C; then
```
Comment (nikhil-kalbande) [inline g/glassfish/glassfish_ubi_9.3.sh:55]: @ramnathnayak-ibm Can's we separate buiild and test part in script

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: g/glassfish/glassfish_ubi_9.3.sh (hunk 1 above)
Comment (ramnathnayak-ibm) [reply g/glassfish/glassfish_ubi_9.3.sh:55]: @nikhil-kalbande Some of the tests are failing for this. I will try to skip those tests and add test part in the script.

File: g/glassfish/glassfish_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
..., !appserver/admin/template, !appserver/ejb/ejb-timer-databases, !appserver/connectors/descriptors, !appserver/jms/jmsra, !nucleus/glassfish-jul-extension, !appserver/jdbc/jdbc-ra/jdbc-ra-distribution, !appserver/persistence/cmp/cmp-scripts, !appserver/batch/batch-database, !appserver/extras/jakartaee/dist-frag, !appserver/extras/appserv-rt/dist-frag, !appserver/grizzly/glassfish-grizzly-extra-all, !appserver/webservices/webservices-scripts, !appserver/appclient/client/appclient-scripts, !appserver/webservices/metro-fragments, !appserver/distributions/glassfish-common, !appserver/distributions/glassfish, !appserver/extras/embedded/common/bootstrap, !appserver/extras/embedded/shell/glassfish-embedded-shell-frag, !appserver/extras/embedded/shell/glassfish-embedded-static-shell-frag, !appserver/extras/embedded/all, !appserver/extras/embedded/web, !appserver/distributions/web, !appserver/tests/admin, !appserver/tests/admin/tests, !appserver/tests/appserv-tests/lib, !appserver/tests/appserv-tests, !appserver/tests/appserv-tests/connectors-ra-redeploy/rars, !appserver/tests/appserv-tests/connectors-ra-redeploy/rars-xa, !appserver/tests/embedded/scatteredarchive, !docs/distribution'; then
```
Comment (nikhil-kalbande) [inline g/glassfish/glassfish_ubi_9.3.sh:62]: @ramnathnayak-ibm Are these failing for both intel and power?

Comment (ramnathnayak-ibm) [reply g/glassfish/glassfish_ubi_9.3.sh:62]: Yes

**Review by nikhil-kalbande — APPROVED** [review]

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**